code/game/machinery/hologram.dm

```diff
@@ -58,10 +58,24 @@ var/global/const/HOLOPAD_MODE = RANGE_BASED
 	var/allow_ai = TRUE
 	var/static/list/reachable_overmaps = list(OVERMAP_ID_SPACE)
 
+	var/holopad_id
+
 /obj/machinery/hologram/holopad/Initialize()
 	. = ..()
-	var/area/A = get_area(src)
-	desc = "It's a floor-mounted device for projecting holographic images. Its ID is '[A.proper_name]'"
+
+	// Null ID means we want to use our area name.
+	global.holopads += src
+	if(isnull(holopad_id))
+		var/area/A = get_area(src)
+		holopad_id = A?.proper_name || "Unknown"
+
+	// For away sites, always tag the sector name so we have a unique discriminator for long range calls.
+	var/obj/effect/overmap/visitable/sector = global.overmap_sectors[num2text(z)]
```

Comment:
Is it guaranteed that the main map doesn't have a sector/that this will only apply to away sites, as stated in the comment? (I have a map downstream that does, so if that's not allowed I need to fix that.)

Reply:
The variable is named sector, I didn't realize. I thought it was the `/obj/effect/overmap/visitable/sector` type. Whoops.